ui/src/components/ui/button.tsx

```diff
@@ -6,13 +6,15 @@ import { cn } from '@/components/ui/utils';
 
 const buttonVariants = cva(
     `inline-flex items-center justify-center whitespace-nowrap rounded-[0.25rem] text-base font-normal ring-offset-white
-        transition-colors ease-in-out duration-150 focus-visible:outline-none focus-visible:ring-2 
+        transition-color ease-in-out duration-150 focus-visible:outline-none focus-visible:ring-2 
```

Comment:
My mistake, I gave you code that isn't correct. It should be `transition-colors` instead of `transition-color`